### PR TITLE
Fixes task #5245 to add ReLU activation frontend tensorflow and merged latest commit histories

### PR DESCRIPTION
**Pull request by suvadityamuk**
#5245

### CODE REVIEW

**Comment by suvadityamuk** [comment]
Hi, @JonathanYang-yzh @jkeane508 Could y'all please check this one out? Thanks!

File: ivy/functional/frontends/tensorflow/activations.py [hunk 1]
```diff
@@ -15,6 +15,9 @@ def hard_sigmoid(x):
 def linear(x):
     return x
 
+def relu(x):
+    x = ivy.clip(x, 0.0, ivy.inf)
```
Comment (jkeane508) [inline ivy/functional/frontends/tensorflow/activations.py:19]: I've just checked the docs and there is actually an `ivy.relu()` function in the [API](https://lets-unify.ai/ivy/functional/ivy/activations/relu/relu_functional.html?highlight=relu#ivy-relu), you can just use it instead :)

**Review by jkeane508 — CHANGES_REQUESTED** [review]

**Review by jkeane508 — CHANGES_REQUESTED** [review]
There are also some lint errors with this file, please install a [pre-commit hook](https://lets-unify.ai/ivy/contributing/0_setting_up.html#pre-commit) to fix these before pushing to your branch :)

**Comment by suvadityamuk** [comment]
Hi, I've corrected the file and set up the linting as well. Please check it now

**Review by jkeane508 — APPROVED** [review]
LGTM!

**Comment by suvadityamuk** [comment]
Awesome, thanks!

**Pull request merged**